<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>DataView</class>
    <widget class="QWidget" name="DataView">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>400</width>
                <height>300</height>
            </rect>
        </property>
        <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
            </sizepolicy>
        </property>
        <property name="minimumSize">
            <size>
                <width>400</width>
                <height>300</height>
            </size>
        </property>
        <property name="baseSize">
            <size>
                <width>400</width>
                <height>300</height>
            </size>
        </property>
        <property name="windowTitle">
            <string>JSON Viewer</string>
        </property>
        <widget class="QLabel" name="label">
            <property name="geometry">
                <rect>
                    <x>30</x>
                    <y>30</y>
                    <width>80</width>
                    <height>16</height>
                </rect>
            </property>
            <property name="text">
                <string>Some Stuff</string>
            </property>
        </widget>
        <widget class="QLineEdit" name="lineEdit_2">
            <property name="geometry">
                <rect>
                    <x>230</x>
                    <y>190</y>
                    <width>113</width>
                    <height>20</height>
                </rect>
            </property>
        </widget>
        <widget class="QLineEdit" name="lineEdit_3">
            <property name="geometry">
                <rect>
                    <x>60</x>
                    <y>200</y>
                    <width>113</width>
                    <height>20</height>
                </rect>
            </property>
        </widget>
        <widget class="QGroupBox" name="groupbox">
            <property name="geometry">
                <rect>
                    <x>120</x>
                    <y>10</y>
                    <width>271</width>
                    <height>121</height>
                </rect>
            </property>
            <property name="title">
                <string>Some Other Stuff</string>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout">
                <item>
                    <widget class="QLineEdit" name="lineEdit"/>
                </item>
                <item>
                    <widget class="QLabel" name="label_2">
                        <property name="geometry">
                            <rect>
                                <x>30</x>
                                <y>30</y>
                                <width>80</width>
                                <height>16</height>
                            </rect>
                        </property>
                        <property name="text">
                            <string>Some Stuff</string>
                        </property>
                    </widget>
                </item>
                <item>
                    <spacer name="horizontalSpacer">
                        <property name="orientation">
                            <enum>Qt::Horizontal</enum>
                        </property>
                        <property name="sizeHint" stdset="0">
                            <size>
                                <width>40</width>
                                <height>20</height>
                            </size>
                        </property>
                    </spacer>
                </item>
                <item>
                    <widget class="QLCDNumber" name="qlcdnumber">
                        <property name="geometry">
                            <rect>
                                <x>30</x>
                                <y>30</y>
                                <width>60</width>
                                <height>30</height>
                            </rect>
                        </property>
                        <property name="smallDecimalPoint">
                            <bool>true</bool>
                        </property>
                        <property name="value">
                            <double>3.14</double>
                        </property>
                    </widget>
                </item>
                <item/>
            </layout>
        </widget>
        <widget class="QRadioButton" name="radioButton">
            <property name="geometry">
                <rect>
                    <x>10</x>
                    <y>70</y>
                    <width>82</width>
                    <height>17</height>
                </rect>
            </property>
            <property name="text">
                <string>Station 1</string>
            </property>
        </widget>
        <widget class="QRadioButton" name="radioButton_2">
            <property name="geometry">
                <rect>
                    <x>10</x>
                    <y>100</y>
                    <width>82</width>
                    <height>17</height>
                </rect>
            </property>
            <property name="text">
                <string>Station 2</string>
            </property>
            <property name="checkable">
                <bool>true</bool>
            </property>
            <property name="checked">
                <bool>true</bool>
            </property>
        </widget>
        <widget class="QSpinBox" name="spinBox">
            <property name="geometry">
                <rect>
                    <x>30</x>
                    <y>150</y>
                    <width>42</width>
                    <height>22</height>
                </rect>
            </property>
        </widget>
        <widget class="QDoubleSpinBox" name="doubleSpinBox">
            <property name="geometry">
                <rect>
                    <x>80</x>
                    <y>150</y>
                    <width>49</width>
                    <height>20</height>
                </rect>
            </property>
            <property name="wrapping">
                <bool>true</bool>
            </property>
            <property name="frame">
                <bool>true</bool>
            </property>
            <property name="readOnly">
                <bool>false</bool>
            </property>
            <property name="buttonSymbols">
                <enum>QAbstractSpinBox::UpDownArrows</enum>
            </property>
            <property name="minimum">
                <double>0</double>
            </property>
            <property name="maximum">
                <double>115.9</double>
            </property>
            <property name="singleStep">
                <double>1.1</double>
            </property>
            <property name="value">
                <double>98.3</double>
            </property>
        </widget>
    </widget>
    <resources/>
    <connections/>
</ui>
